trurl.yaml

```
@@ -0,0 +1,45 @@
+package:
+  name: trurl
+  version: '0.10'
+  epoch: 0
+  description: 'trurl is a command line tool for URL parsing and manipulation.'
+  copyright:
+    - license: curl License
```

https://spdx.org/licenses/curl.html we use the short identifier 
```suggestion
    - license: curl
```

---

fixed